frontend/src/views/Collection/index.tsx

```diff
@@ -70,19 +108,23 @@ const Collection: FC = () => {
           <H3>{collection.name}</H3>
           <Description>{collection.description}</Description>
           <LinkContainer>
-            {renderLinks(collection.links)}
+            {/*
+             * (thuang): Contact and Links order defined here:
+             * https://app.zenhub.com/workspaces/single-cell-5e2a191dad828d52cc78b028/issues/chanzuckerberg/single-cell/124
+             */}
             {renderContact(collection.contact_name, collection.contact_email)}
+            {renderLinks(collection.links)}
           </LinkContainer>
         </CollectionInfo>
 
-        <CollectionButtons>
-          {collection.access_type === ACCESS_TYPE.WRITE && isPrivate && (
-            <DeleteCollection id={id} />
-          )}
-          {isPrivate && (
-            <PublishCollection isPublishable={isPublishable} id={id} />
-          )}
-        </CollectionButtons>
+        {shouldShowPrivateWriteAction && (
```

Comment:
We hide all the private write action buttons inside `<ActionButtons />` with `shouldShowPrivateWriteAction` guard